share/spack/gitlab/cloud_pipelines/stacks/build_systems/spack.yaml

```diff
@@ -50,9 +103,9 @@ spack:
         - . "./share/spack/setup-env.sh"
         - spack --version
       image: { "name": "ghcr.io/spack/e4s-ubuntu-18.04:v2021-10-18", "entrypoint": [""] }
-      tags: ["spack", "public", "medium", "x86_64"]
+      tags: ["spack", "public", "default", "x86_64"]
 
-  cdash:
```

Comment:
Removal of this space is what I think caused the failure of the generation job for this stack, [here](https://gitlab.spack.io/spack/spack/-/jobs/2247448).

Reply:
It was clearer in the diff view, here not so much, but a space was removed before the `cdash`.